addons/web_tour/static/src/js/tip.js

```diff
@@ -272,11 +271,17 @@ var Tip = Widget.extend({
 
         return $location;
     },
-    _reposition: function () {
+    /**
+     * Repositions the tip.
+     *
+     * @param {String} [forcePosition] force a position instead of 'this.info.position'
+     *   Typically useful when forcing a temporary position because of lack of viewport space.
+     */
+    _reposition: function (forcePosition) {
```

Comment:
forcePosition in the above function is a boolean, so this is a bit misleading. I would simply name it `position` here